UPGRADE-GUIDE.md

```diff
@@ -0,0 +1,465 @@
+# Ganache Upgrade Guide
+
+## ganache-core v2.x.x and ganache-cli v6.x.x to ganache v7.x.x
+
+### Installing
+
+The Ganache npm packages have been combined and renamed from `ganache-cli` and
+`ganache-core` to `ganache`. You may need to uninstall the old packages before
+installing the new ganache.
+
+There are two ways you may have the old packages installed, globally or locally.
+The uninstall and installation process are slightly different for each.
+
+#### Globally installed
+
+A global installation makes it possible to run Ganache from your command line at
+any working directory: `$~ ganache [...options]`.
+
+To upgrade a global installation using [`npm`](____) run these two commands:
+
+```console
+$ npm uninstall --global ganache-cli ganache-core
+$ npm install --global ganache
+```
+
+_For security reasons[^1] we do not recommend using yarn or pnpm as your package
+manager when installing security-critical applications like Ganache._
+
+You will now be able to run `$~ ganache` from your command line.
+
+_note: you may need to uninstall using yarn, or another node package manager,
```

Comment:
```suggestion
_NOTE: you may need to uninstall using yarn, or another node package manager,
```
Capitalizing NOTEs to be consistent across this doc and with other ganache READMEs. Also consider un-italicizing for consistency, unless the intent is to bring extra emphasis to these notes.